# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/screen-capture/getdisplaymedia.https.html.ini
# Commit: 85883a720dab
# Full Hash: 85883a720dab61e82bcdb7391236dc06427466ee
# Author: moz-wptsync-bot <wptsync@mozilla.com>
# Date: 2021-04-07 15:47:02
# Regressor Bug: 1702830
# File Overlap Count: 1
# Description:
#   Bug 1702830 - [wpt-sync] Update web-platform-tests to 336ec39f06d15863a4af6b2aa6592530b37a5aee, a=testonly
#   
#   MANUAL PUSH: wpt sync bot
#   
#   wpt-head: 336ec39f06d15863a4af6b2aa6592530b37a5aee
# ==============================================================================

diff -r acef12d74007 -r 85883a720dab testing/web-platform/meta/screen-capture/getdisplaymedia.https.html.ini
--- a/testing/web-platform/meta/screen-capture/getdisplaymedia.https.html.ini	Fri Apr 02 22:03:46 2021 +0000
+++ b/testing/web-platform/meta/screen-capture/getdisplaymedia.https.html.ini	Fri Apr 02 22:06:52 2021 +0000
@@ -3,8 +3,10 @@
     if (os == "android") and e10s: bug 1550895 (frequently fails on geckoview)
   expected:
     if (processor == "x86_64") and not swgl and (os == "win") and debug and webrender: ["OK", "CRASH"]
+    if (processor == "x86") and (os == "win") and debug: ["OK", "CRASH"]
+    if (processor == "x86") and (os == "win") and not debug: ["CRASH", "OK"]
     if (processor == "x86_64") and swgl and (os == "win"): ["OK", "CRASH"]
-    if processor == "x86": ["OK", "CRASH"]
+    if (processor == "x86") and (os == "linux"): ["OK", "CRASH"]
   [getDisplayMedia() with getSettings]
     expected: FAIL
 